### PR TITLE
Lint tests

### PR DESCRIPTION
**Pull request by phapdinh**
Hi @jdalton, I thought I would help with code consistency between your test files and the other lodash files by adding the test/*.test.js to your linting in npm run style. Most of this was done by eslint with the --fix flag. I just added mocha and _ to the configurations. I ran npm run test:wip and it passes. It is up to you if you want the feature, but I thought I put it out there.

### CODE REVIEW

**Comment by blikblum** [comment]
IMO, in this moment, it should not bother with test files because many of these files are not relevant anymore and will surely be removed in future

**Comment by phapdinh** [comment]
I see the *.test.js files being updated and recent pull requests so I thought they were going to be used in the future, but I might be wrong

File: .eslintrc.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ module.exports = {
   'plugins': ['import'],
   'env': {
     'es6': true,
-    'node': true
+    'node': true,
+    'mocha': true
+  },
+  'globals': {
+    '_': 'readonly'
```
Comment (armano2) [inline .eslintrc.js:10]: you should all those env variables coditionally only if file is `*.test.js`
```js
overrides: [
  // all test files
  {
      files: ['**/*.test.js'],
      env: {
        'mocha': true
      },
      'globals': {
         '_': 'readonly'
      ]
  }
]
```

**Comment by phapdinh** [comment]
too many conflicts

**Pull request closed without merging**